.github/workflows/lokalise-automerger.yml

```
@@ -0,0 +1,71 @@
+name: Lokalise PR validation, auto approve and merge
+on:
+  pull_request:
+    branches:
+      - master
+      - develop
+    paths:
+      - 'src/locale/*.json'
+jobs:
+  autocheckandapprove:
+    runs-on: ubuntu-latest
```

about the following: 
>You'll also see that in each step of the action this if statement is added:
if: github.actor == 'VWSCoronaDashboard' && startsWith(github.head_ref, 'lokalise-') == true
Again ideally, I'd like to be able to perform such a check at the start of a job and be able to end the job when the condition is not met. I have not found a clean way of actually doing this. Therefore the check is added to each step. Any suggestions here are welcome too.

Doesn't [`jobs.<job_id>.if`](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idif) do the job?

```suggestion
jobs:
  autocheckandapprove:
    if: github.actor == 'VWSCoronaDashboard' && startsWith(github.head_ref, 'lokalise-') == true
    runs-on: ubuntu-latest
```

---

hahaha, well, duh, that sounds very likely indeed. I completely missed that one! I am going to test it out a.s.a.p.! Thanks!